rabbitmq_queue_master_balancer/rabbitmq_queue_master_balancer.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python
+
+import logging
+import time
+
+from configargparse import ArgumentParser
+from prettylog import basic_config, LogFormat
+from pyrabbit2.api import Client
+from pyrabbit2.http import NetworkError
+
+parser = ArgumentParser(auto_env_var_prefix='RQMB_')
+
+parser.add_argument(
+    '--api-url',
+    type=str,
+    default='localhost:15672'
+)
+
+parser.add_argument(
+    '--user',
+    type=str,
+    default='guest'
+)
+
+parser.add_argument(
+    '--password',
+    type=str,
+    default='guest'
+)
+
+parser.add_argument(
+    '-L', '--log-level', help='Log level',
+    default='info',
+    choices=(
+        'critical', 'fatal', 'error', 'warning',
+        'warn', 'info', 'debug', 'notset'
+    ),
+)
+
+parser.add_argument(
+    '--log-format', choices=LogFormat.choices(), default=LogFormat.stream
+)
+
+parser.add_argument(
+    '--dry-run',
+    action="store_true",
+    help='Show how many queues will move from one node (with a maximum queues) \
```

Comment:
do not split it with backslash, just use:

```python

(
  'long'
  'string'
)
```

Reply:
+